### ISSUE TITLE
Tesseract fails to detect letters Å and å in Finnish language.

### ISSUE THREAD

**Issue by jmokoistinen** [tesseract-ocr/langdata_lstm#31]
Testing Tesseract to detect Finnish texts containing "swedish o" -> å. Seems it cannot detect them- Å and å correctly. I have also tried fin+swe model but more usually the fin model version of the text is selected. 

Is the previous training files available somewhere? Probably the training data does not have enough Åå cases or it is not included even it is official letter.

**Comment by jmokoistinen** [comment]
Yes, what should i do to make it happen? Collect some data and box them with some tool? where can i get the current data? Cannot see any images here https://github.com/tesseract-ocr/langdata_lstm/tree/master/fin

**Comment by jmokoistinen** [comment]
Also letters Q and q are missing from the data? There should be all letters at least abcdefghijklmnopqrstuvwxyzåäö 
ABCDEFGHIJKLMNOPQRSTUVWXYZÅÄÖ
1234567890
How can these be fixed?

**Comment by stweil** [comment]
I'd add all desired characters to `desired_characters`, ideally sorted with `LANG=C.UTF-8 sort`. Then we at least have a list of those characters and can try to find training texts which include them sufficiently often.

To fix the problem, we still have to run new training ...
